posthog/test/test_user_permissions.py

```diff
@@ -45,6 +45,22 @@ def test_team_effective_membership_level_does_not_belong(self):
         with self.assertNumQueries(1):
             assert permissions.team(self.team).effective_membership_level is None
 
+    def test_team_effective_membership_level_membership_isolation(self):
```

Comment:
ah, that's here 😊 

Should this be in the EE test file too? Or comments linking them?

Reply:
Ah yeah ExplicitTeamMembership is an EE class so the tests should probably be there?

Reply:
Hmm, perhaps. Though we've been fine with using these tests here for a while, so not gonna rock the boat for the time being.